components/common/Text.tsx

```diff
@@ -0,0 +1,46 @@
+import { useFonts } from 'expo-font';
+import { useContext } from 'react';
+import { StyleProp, StyleSheet, Text, TextStyle } from 'react-native';
+import { FontLoadedContext } from '../../App';
+import TitleSizes from '../../theme/fontSizes';
+
+interface TextProps {
+  children: React.ReactNode;
+  size?: number;
+  style: StyleProp<TextStyle>;
+  centered?: boolean;
+}
+
+const LText = ({ children, size = 12, style, centered }: TextProps) => {
+  const textAlign = centered ? 'center' : 'left';
+  const fontsLoaded = useContext(FontLoadedContext);
+  let currentStyle = styles.titleTextCustom;
+
+  if (!fontsLoaded) {
+    currentStyle = styles.titleText;
+    return <Text style={[currentStyle, { fontSize: size, textAlign }, style]}>{children}</Text>;
```

Comment:
You can remove the return in if statement, it is redundant. We return the same thing below anyways, setting `currentStyle` variable in the if statement seems enough.

Reply:
bunu circle oluyo diye warning verdiği için eklemiştim bi şey değişmedi ama. En son böyle unutmuşum